src/publisher/aws_events.py

```diff
@@ -1,52 +1,108 @@
+from ordered_set import OrderedSet
 import json
 from django.conf import settings
 import boto3
 from publisher import relation_logic as relationships
-from publisher.utils import lmap
+from functools import wraps
+
+from multiprocessing import SimpleQueue
```

Comment:
`SimpleQueue` is for multiple threads writing to the same data structure I think. Why not just `Queue.Queue`?